src/main/java/DukeComponent/Ui.java

```diff
@@ -0,0 +1,71 @@
+package DukeComponent;
+
+import TaskList.TaskList;
+import Tasks.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Components.Ui contains all the responses the program can give to a command.
+ */
+public class Ui {
+    private static String print(String s) {
+        return s;
+    }
+
+    public static String welcomeMessage() {
```

Comment:
Should the method be named more accurately with a verb, perhaps `getWelcomeMessage()`?